packages/compat/src/compat-addons.ts

```diff
@@ -35,7 +35,12 @@ export default class CompatAddons implements Stage {
 
   async ready(): Promise<{ outputPath: string }> {
     return {
-      outputPath: resolve(locateEmbroiderWorkingDir(this.compatApp.root), 'rewritten-app'),
+      outputPath: resolve(
+        locateEmbroiderWorkingDir(this.compatApp.root),
+        'rewritten-app',
+        'node_modules',
```

Comment:
This should get backed out of the PR. Since you didn't need the fake target for the app, we also don't need to put the app inside this structure.

This will mean a bunch of test changes in the PR also won't be needed.